test/parallel/test-wrap-js-stream-read-stop.js

```diff
@@ -0,0 +1,41 @@
+// Flags: --expose-internals
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const WrappedStream = require('internal/wrap_js_stream');
```

Comment:
Could this be renamed to `StreamWrap` for consistency with the rest of the codebase?

Reply:
Done